react/typings/vtex.render-runtime.d.ts

```diff
@@ -56,7 +56,9 @@ declare module 'vtex.render-runtime' {
     production: boolean
     publicEndpoint: string
     renderMajor: number
-    query?: Record<string, string>
+    query: {
+      q?: string
+    }
```

Comment:
I wouldn't do that, since now every time you want to use `query` attribute from  `render-runtime` that is not `q`, it would not let you. `Record<string, string>` is more accurate